### PR TITLE
WEBAPP-325: Fix url encoding

### PR DESCRIPTION
**Pull request by maxammann**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: src/modules/endpoint/models/CategoriesMapModel.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ class CategoriesMapModel {
    * @return {CategoryModel | undefined} The category
    */
   findCategoryByPath (path: string): ?CategoryModel {
-    return this._categories.get(normalizePath(path))
+    return this._categories.get(decodeURIComponent(normalizePath(path)))
```
Comment (Schedulaar) [inline src/modules/endpoint/models/CategoriesMapModel.js]: should we just use these globally defined functions like `decodeURIComponent` or should these be wrapped in a module (for separation / testability purposes)?

Is there an eslint rule for things like that?

Comment (maxammann) [reply src/modules/endpoint/models/CategoriesMapModel.js]: What do you mean wrapped in a module? Define a composition of  those two methods?

I don't see advantages of a separate function module.

Comment (Schedulaar) [reply src/modules/endpoint/models/CategoriesMapModel.js]: Hmmm don't know^^ I am just a little bit surprised, that we just use globally defined functions, but I think that's alright^^

Comment (maxammann) [reply src/modules/endpoint/models/CategoriesMapModel.js]: Global functions are perfectly fine as long as they do not have side effects :)

Comment (Schedulaar) [reply src/modules/endpoint/models/CategoriesMapModel.js]: Like `fetch` :P

**Comment by Schedulaar** [comment]
This wants to be merged still :D

**Pull request merged**